yt_idefix/fields.py

```diff
@@ -47,3 +62,266 @@ def setup_fluid_fields(self):
+        ("rho", (_rho_units, ["density", "rho"], None)),
+        ("vx1", (_vel_units, ["vel_x"], None)),
+        ("vx2", (_vel_units, ["vel_y"], None)),
+        ("vx3", (_vel_units, ["vel_z"], None)),
+        ("prs", (_pres_units, ["prs", "pres", "pressure"], None)),
+        ("Bx1", ("code_magnetic", [], None)),
+        ("Bx2", ("code_magnetic", [], None)),
+        ("Bx3", ("code_magnetic", [], None)),
+    )
+
+    known_particle_fields = ()
+
+    def setup_fluid_fields(self):
+        unit_system = self.ds.unit_system
+
+        setup_magnetic_field_aliases(
+            self, self.ds._dataset_type, [f"Bx{idir}" for idir in "123"]
+        )
+        for idir in "123":
+            self.alias(
+                ("gas", f"BX{idir}"),
+                (self.ds._dataset_type, f"Bx{idir}"),
+                units="code_magnetic",
+            )
+
+        # Add tracer fields
+        for i in range(1, self.ds.ntracers + 1):
+            if (self.ds._dataset_type, "tr%d" % i) in self.field_list:
+                self.add_output_field(
+                    (self.ds._dataset_type, "tr%d" % i),
+                    sampling_type="cell",
+                    units="",
+                )
+                self.alias(
+                    ("gas", "Tracer %d" % i),
+                    (self.ds._dataset_type, "tr%d" % i),
+                    units="",
+                )
+                self.alias(
+                    ("gas", "tracer %d" % i),
+                    (self.ds._dataset_type, "tr%d" % i),
+                    units="",
+                )
+
+        if (self.ds._dataset_type, "Temp") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "Temp"),
+                sampling_type="cell",
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "Temperature"),
+                (self.ds._dataset_type, "Temp"),
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                (self.ds._dataset_type, "Temp"),
+                units=unit_system["temperature"],
+            )
+        elif (self.ds._dataset_type, "temp") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "temp"),
+                sampling_type="cell",
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "Temperature"),
+                (self.ds._dataset_type, "temp"),
+                units=unit_system["temperature"],
+            )
+            self.alias(
+                ("gas", "temperature"),
+                (self.ds._dataset_type, "temp"),
+                units=unit_system["temperature"],
+            )
+        elif (self.ds._dataset_type, "Temperature") in self.field_list:
+            self.add_output_field(
+                (self.ds._dataset_type, "Temperature"),
```

Comment:
```suggestion
                (self.ds._dataset_type, "temperature"),
```

Reply:
The temperature field, these are commonly defined as either "Temp" or "Temperature" by users in PLUTO. So I'll change this accordingly and reduce a lot of these lines.